types/artifacts.d.ts

```diff
@@ -289,6 +289,9 @@ declare global {
         href: string
         text: string
         target: string
+        path: string
```

Comment:
in scripts gatherer (one of the other "public" gatherers) we named this `devtoolsNodePath` to make it clear this isn't for general use like the selector.